doc/rst/tools/protoc-gen-chpl/protoc-gen-chpl.rst

```diff
@@ -0,0 +1,213 @@
+===============
+Chapel Protobuf
+===============
+
+Chapel Protobuf is a Google Protocol Buffers implementation for Chapel.
+`Protocol Buffers`_ is a language-neutral, platform-neutral, extensible mechanism
+for serializing structured data. The protocol buffer language is a language for 
+specifying the schema for structured data. This schema is compiled into language
+specific bindings.
+
+This project is comprised of two components:
+
+* Code generator: The ``protoc-gen-chpl`` tool is a compiler plugin to ``protoc``, the protocol
+  buffer compiler. It enables the ``protoc`` compiler to generate Chapel code.
+  
+* Chapel library: A module that has the runtime implementation of protobufs
+  in Chapel and provides functionality to serialize messages in wire format.
+
+
+Installation Instructions
+=========================
+
+You should have the protocol buffer compiler installed on your system. For example,
+to install protobuf on an Ubuntu system, you might use ``apt-get install libprotobuf-dev``.
```

Comment:
I needed to install also libprotoc-dev protobuf-compiler to get it to build

Reply:
@mppf Should we rely on `Debian` packages? The download section of the official `protobuf` guide also points to the source repository instead of `debian` packages.

Reply:
Since the packages did not work for me at all - I think this should suggest installing 3.13 or later - and note that source installation might currently be required. It doesn't make sense to talk about `apt-get` in that event.